src/atoms/Space/__tests__/Space.native.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import { View } from 'react-native';
+
+import Space from '../index';
+
```

Comment:
the test is also missing for the case where it will throw an error 🙈

Reply:
Added